_build/Extension-SetIdentityAndVersion.ps1

```diff
@@ -0,0 +1,198 @@
+      Write-Host "Applying Display Name to vsixlangpack files"
+      foreach ($langPack in $vsixLangPacks) {
+        [xml]$langContent = Get-Content $langPack
+        $langContent.VsixLanguagePack.LocalizedName = $vsixDisplayName
+        $langContent.Save($langPack) 
+        Write-Host "$langPack - LocalizedName applied ($vsixDisplayName)"        
+      }
+    }
+  }
+  else{
+    throw "No VSIX manifest file found."
+  }
+}
+else{
+  throw "Identity is mandatory."
+}
+
+
+## REPLACE Command Guids
+if($cmdSetGuid -and $packageGuid){
+  Write-Host
+  Write-Host "Setting PackageGuid and CmdSetGuid in VSCT Files"
+  $vsctFiles = Get-ChildItem -include "*.vsct" -recurse |  Where-Object{ 
+      $_.FullName -notmatch "\\Templates\\" -and 
+      $_.FullName -notmatch "\\debug\\" -and
+      $_.FullName -notmatch "\\obj\\" -and
+      $_.FullName -match "\\RapidXamlToolkit\\"
+  }
+  if($vsctFiles){ 
+    Write-Host
+    Write-Host "Applying guids $cmdSetGuid, $packageGuid and command name $vsixCommandMenuName to VSCT Files"
+    foreach ($vsctFile in $vsctFiles) {
+      $vsctFileContent = Get-Content $vsctFile
+      attrib $vsctFile -r
+      $replacedVsctContent = $vsctFileContent.Replace($targetPackageGuid, $packageGuid)
+      $replacedVsctContent = $replacedVsctContent.Replace($targetCmdSetGuid, $cmdSetGuid)
+      $replacedVsctContent = $replacedVsctContent.Replace($targetVsixCommandMenuName, $vsixCommandMenuName)
+      $replacedVsctContent | Out-File $vsctFile -Encoding utf8 -Force
+      Write-Host "$vsctFile - Guids applied (PackageGuid:$packageGuid, CmdGuid:$cmdSetGuid)"        
+    }
+  }
+  else{
+    throw "No VSCT files found."
+  }
+
+  Write-Host
+  Write-Host "Applying guids $packageGuid to RapidXamlPackage.cs"
+  $path = Resolve-Path $vsixManifestFile
+  $installerPath = Split-Path $path
+  $constFile = Join-Path  $installerPath "RapidXamlPackage.cs"
+  
+  if($constFile){
+    $constFileContent = Get-Content $constFile
+    attrib $constFile -r
+    $replacedConstContent = $constFileContent.Replace($targetPackageGuid, $packageGuid)
+    $replacedConstContent | Out-File $constFile -Encoding utf8 -Force
+    Write-Host "$constFile - Guid applied (PackageGuid:$packageGuid)"
+  }
+  else{
+     throw "BaseCommand.cs constants file not found"
+  }
+
+  Write-Host
+  Write-Host "Applying guids $cmdSetGuid to BaseCommand.cs"
+  $path = Resolve-Path $vsixManifestFile
+  $installerPath = Split-Path $path
+  $constFile = Join-Path  $installerPath "Commands\BaseCommand.cs"
+  
+  if($constFile){
+    $constFileContent = Get-Content $constFile
+    attrib $constFile -r
+    $replacedConstContent = $constFileContent.Replace($targetCmdSetGuid, $cmdSetGuid)
+    $replacedConstContent | Out-File $constFile -Encoding utf8 -Force
+    Write-Host "$constFile - Guid applied (CmdGuid:$cmdSetGuid)"
+  }
+  else{
+     throw "BaseCommand.cs constants file not found"
+  }
+}
+else{
+  throw "PackageGuid and CmdSetGuid are mandatory"
```

Comment:
Wouldn't this be caught when the param is not passed, as it's marked as mandatory?

Reply:
oops. marked wrong line